policies/release-requirements.md

```diff
@@ -0,0 +1,137 @@
+Release Requirements Policy
+===========================
+
+The OpenSSL project team creates the following 5 types of OpenSSL software
+releases:
+
+- [alpha] (pre-)releases
+- [beta] (pre-)releases
+- [major] and [minor] releases
```

Comment:
These should be broken into two separate lines.
- [major] releases
- [minor] releases